src/utils/immutableUtils.js

```diff
@@ -19,3 +19,16 @@ export function imDeleteFromArrayById(array = [], idFieldName, idValue) {
 
   return imDeleteFromArray(array, indexToDelete);
 }
+
+export function imSetToArray(array = [], mapToAdd = {}, objUniqueField = '', index) {
```

Comment:
Please use full names 😢 `im` is not readable

Reply:
[Fixed](https://github.com/LiskHQ/lisk-desktop/pull/4912/commits/3fd8cc9f2e2974151935fd3be5546e48f9761a80)